classes/Form.php

```diff
@@ -915,6 +915,11 @@ public function post()
                     $data = $data[$action];
                 }
 
+                // do not execute action, if deactivated
+                if (in_array($data, [false, 'false'], true)) {
```

Comment:
Using `Utils::isNegative($data)` might be better idea? The class is in `\Grav\Common` namespace.

Reply:
Not a good idea after all. This is because any of those values may be valid input for the process. This includes 'false'...

I would at most test:
```
if ($data === false) {
```

If we can assume that false is never going to be used in any other meaning.

Reply:
I checked admin, login, form, and email plugins, they look good (if only checking boolean false). 

@rhukster What do you think?

Reply:
@mahagr Do you want me to change it to check only boolean value?